helpers/setVarBlock.js

```diff
@@ -0,0 +1,9 @@
+function setVar(varName, options) {
```

Comment:
I'm not quite sure what the purpose of this is? For the modelIncludes file, I would assume that we're trying to achieve

commonjs:
```
const {
  // ...
} = require("...");
```
What am I missing here?

es6:
```
import {
  // ...
} from "...";
```

Reply:
Yes, that is what we're looking to achieve. The `import` handlebars helper generates the boilerplate you detailed. However, you need to pass in the names of the objects to import. If the name is something that is constructed dynamically by the template we need a way to capture that.